### PR TITLE
cassandra driver bump

### PR DESCRIPTION
**Pull request by rortian**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+source 'https://rubygems.pkg.github.com/art19'
```
Comment (tatthurs) [inline Gemfile]:
```suggestion

# pull the ART19 patched version of cassandra-driver
source 'https://rubygems.pkg.github.com/art19' do
  gem 'cassandra-driver', '~> 3.5.2.1'
end
```

File: cequel.gemspec [hunk 2]
```diff
@@ -29,7 +29,7 @@ DESC
 
   s.add_runtime_dependency 'activemodel', '>= 4.0'
   s.add_runtime_dependency 'bigdecimal', '~> 3.0'
-  s.add_runtime_dependency 'cassandra-driver', '~> 3.0'
+  s.add_runtime_dependency 'cassandra-driver', '~> 3.5.2.1'
```
Comment (tatthurs) [inline cequel.gemspec]:
```suggestion
  # Require the ART19 patched version of cassandra-driver
  s.add_runtime_dependency 'cassandra-driver', '3.5.2.1'
```

File: Gemfile (hunk 1 above)
Comment (rortian) [reply Gemfile]: No opposed, I did think it would be weird given the gemspec import

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -51,3 +53,4 @@ jobs:
           bundle exec rake test
         env:
           RAILS_ENV: test
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (tatthurs) [inline .github/workflows/test.yml]:
```suggestion
```

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -42,6 +42,8 @@ jobs:
     env:
       BUNDLE_GEMFILE: gemfiles/rails_${{ matrix.rails }}.gemfile
     steps:
+      - run: |
+          bundle config --local github.com $GH_TOKEN
```
Comment (tatthurs) [inline .github/workflows/test.yml]:
```suggestion
```

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -42,6 +42,8 @@ jobs:
...
       - uses: actions/checkout@v2
       - uses: ruby/setup-ruby@v1
         with:
```
Comment (tatthurs) [inline .github/workflows/test.yml:49]:
```suggestion
        env: BUNDLE_RUBYGEMS__PKG__GITHUB__COM: ${{ secrets.GITHUB_TOKEN }}
        with:
```

File: .github/workflows/test.yml [hunk 6]
```diff
@@ -44,6 +44,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: ruby/setup-ruby@v1
+        env: BUNDLE_RUBYGEMS__PKG__GITHUB__COM: ${{ secrets.GITHUB_TOKEN }}
```
Comment (tatthurs) [inline .github/workflows/test.yml]:
```suggestion
        env: 
          - BUNDLE_RUBYGEMS__PKG__GITHUB__COM: ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/test.yml [hunk 7]
```diff
@@ -44,6 +44,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: ruby/setup-ruby@v1
+        env: 
+          - BUNDLE_RUBYGEMS__PKG__GITHUB__COM: ${{ secrets.GITHUB_TOKEN }}
```
Comment (tatthurs) [inline .github/workflows/test.yml]:
```suggestion
          BUNDLE_RUBYGEMS__PKG__GITHUB__COM: ${{ secrets.GITHUB_TOKEN }}
```

File: cequel.gemspec [hunk 8]
```diff
@@ -29,7 +29,8 @@ DESC
 
   s.add_runtime_dependency 'activemodel', '>= 4.0'
   s.add_runtime_dependency 'bigdecimal', '~> 3.0'
-  s.add_runtime_dependency 'cassandra-driver', '~> 3.0'
+  # Require the ART19 patched version of cassandra-driver
+  s.add_runtime_dependency 'cassandra-driver', '3.2.5.1'
```
Comment (lexahu) [inline cequel.gemspec]: why no ~> here?

Comment (tatethurston) [reply cequel.gemspec:33]:
```suggestion
  s.add_runtime_dependency 'cassandra-driver', '~> 3.2.5.1'
```

Yeah this was me when we were debugging

**Review by tatethurston — APPROVED** [review]

File: cequel.gemspec (hunk 8 above)
Comment (tatthurs) [inline cequel.gemspec]:
```suggestion
  s.add_runtime_dependency 'cassandra-driver', '~> 3.2.5.1'
```
Yeah this was me debugging

**Review by lexahu — APPROVED** [review]

**Pull request merged**